### PR TITLE
improvements related to QS feedback

### PR DESCRIPTION
**Pull request by bhusang**

### CODE REVIEW

File: designs/tutorial.stanza [hunk 1]
```diff
@@ -1,81 +1,90 @@
...
   inst lm317a : ocdb/components/unisonic/LM317A/component
-  net (lm317a.input vin)
-  net (lm317a.output vout)
+  net (lm317a.input, vin)
+  net (lm317a.output, vout)
 
-  cap-strap(vin, gnd, 0.1e-6)
-  cap-strap(vout, gnd, 1.0e-6)
+  cap-strap(vin, gnd, ["capacitance" => 0.1e-6 "min-rated-voltage" => 25.0])
+  cap-strap(vout, gnd, ["capacitance" => 1.0e-6 "min-rated-voltage" => 25.0])
```
Comment (m-hilgendorf) [inline designs/tutorial.stanza:24]: Add commas in tuples.

File: designs/tutorial.stanza [hunk 2]
```diff
@@ -1,81 +1,90 @@
...
+  val true-adj-offset = adj-current * (property(output-voltage-divider.r-lo.resistance) as Double)
   val vref = property(output-voltage-divider.output-voltage)
 
   ; Calculate the possible min and max values for the output voltage
-  val out = min-typ-max(min-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / max-value(vref) + true-adj-offset,
-                        typ-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / typ-value(vref) + true-adj-offset,
-                        max-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / min-value(vref) + true-adj-offset)
+  val out = min-typ-max(min-value(refVoltage) * typ-value(output-voltage) / max-value(vref) + true-adj-offset,
+                        typ-value(refVoltage) * typ-value(output-voltage) / typ-value(vref) + true-adj-offset,
+                        max-value(refVoltage) * typ-value(output-voltage) / min-value(vref) + true-adj-offset)
```
Comment (m-hilgendorf) [inline designs/tutorial.stanza:58]: `ref-voltage` instead of `refVoltage` (stanza prefers `kebab-case` to `camelCase`)

**Review by m-hilgendorf — COMMENTED** [review]
Just a bunch of minor style nits for consistency with the rest of OCDB

File: designs/tutorial.stanza [hunk 3]
```diff
@@ -1,81 +1,90 @@
...
   val divider-current = 5.0e-3
+  val adj-current = property(lm317a.adj-current) ; retrieve the property from the component definition
   ; Estimate offset voltage caused by current sourced from adj pin
-  val est-v-adj-offset = (typ-value(output-voltage) - typ-value(property(lm317a.reference-voltage))) / divider-current * property(lm317a.adj-current)
+  val est-v-adj-offset = (typ-value(output-voltage) - typ-value(refVoltage)) / divider-current * adj-current
 
   inst output-voltage-divider : ocdb/modules/passive-circuits/voltage-divider(high-voltage, adj-voltage, current) where :
     val high-voltage = tol%(center-value(output-voltage) - est-v-adj-offset, 0.0)
-    val adj-voltage = tol%(typ-value(property(lm317a.reference-voltage)),target-variance)
+    val adj-voltage = tol%(typ-value(refVoltage),target-variance)
```
[the remark this answers is not in the document]
Comment (bhusang) [reply designs/tutorial.stanza:42]: fixed.

File: designs/tutorial.stanza [hunk 4]
```diff
@@ -1,81 +1,90 @@
...
-  net (output-voltage-divider.lo gnd)
+  net (output-voltage-divider.in, lm317a.output)
+  net (output-voltage-divider.out, lm317a.adj adj)
+  net (output-voltage-divider.lo, gnd)
+  ; inside pcb-module lm317a-regulator 
   schematic-group(self) = lm317a
 
   ; Calculate the actual offset now that we know the optimized value of the resistors
-  val true-adj-offset = property(lm317a.adj-current) * (property(output-voltage-divider.r-lo.resistance) as Double)
+  val true-adj-offset = adj-current * (property(output-voltage-divider.r-lo.resistance) as Double)
```
[the remark this answers is not in the document]
Comment (bhusang) [reply designs/tutorial.stanza:52]: good suggestion

File: designs/tutorial.stanza (hunk 2 above)
Comment (bhusang) [reply designs/tutorial.stanza:58]: done

**Review by m-hilgendorf — APPROVED** [review]

**Pull request merged**